docs/tutorials/Canopy/soil_canopy_tutorial.jl

```diff
@@ -109,6 +110,67 @@ include(
 # Define the parameters for the soil model and provide them to the model 
 # parameters struct:
 
+# Heterotrophic respiration parameters
+P_sfc = FT(101e3)
+ν = FT(0.556)
+θ_a100 = FT(0.1816)
+D_ref = FT(1.39e-5)
+b = FT(4.547)
+D_liq = FT(3.17)
+α_sx = FT(194e3)
+Ea_sx = FT(61e3)
+kM_sx = FT(5e-3)
+kM_o2 = FT(0.004)
+O2_a = FT(0.209)
+D_oa = FT(1.67)
+p_sx = FT(0.024)
+
+soilco2_type = Soil.Biogeochemistry.SoilCO2Model{FT}
+
+soilco2_ps = SoilCO2ModelParameters{FT}(;
+    P_sfc = P_sfc,
+    ν = ν,
+    θ_a100 = θ_a100,
+    D_ref = D_ref,
+    b = b,
+    D_liq = D_liq,
+    α_sx = α_sx,
+    Ea_sx = Ea_sx,
+    kM_sx = kM_sx,
+    kM_o2 = kM_o2,
+    O2_a = O2_a,
+    D_oa = D_oa,
+    p_sx = p_sx,
+    earth_param_set = earth_param_set,
+);
+
+# soil microbes args
+Csom = (z, t) -> eltype(t)(5.0)
+
+C = FT(100)
+
+soilco2_top_bc = Soil.Biogeochemistry.SoilCO2StateBC((p, t) -> eltype(t)(0.0))
```

Comment:
Could you explain why zero flux is reasonable? thanks!

Reply:
note also Julia's message in the current land chat about specifying Float type. `t` is Float64 (and may in the future be a DateTime), so we cant use eltype(t) anymore. instead we convert it internally (her [PR](https://github.com/CliMA/ClimaLSM.jl/pull/353) takes care of this, but we should try and be consistent about how we handle t in tutorials too

Reply:
I am not yet 100% sure what to do at bottom BC. 
I would need to look at actual data of soil co2 concentration at depth - I wonder if it is very low or very high. 
I don't want to have an infinite source of CO2 from bottom BC (which does happen in some volcanic areas though, but that's the exception). 
And if we say it's 0, we'll have co2 diffusing to below BC constantly, which I imagine would not happen at equilibrium. 

I feel like there must be an equilibrium where diffusion to bottom BC is very small? But I am not sure 100%.

Reply:
ah sorry I thought this was bottom bc. I am changing top to atmospheric C

Reply:
Re: the bottom BC, it's probably OK for now! might be another thing to open an issue on, just to keep track of it?